client/payments/payment-recommendations.tsx

```diff
@@ -123,18 +148,23 @@ const PaymentRecommendations: React.FC = () => {
 		} );
 		installAndActivatePlugins( [ plugin.plugins[ 0 ] ] )
 			.then( () => {
-				window.location.href = getAdminLink(
-					plugin[ 'setup-link' ].replace( '/wp-admin/', '' )
-				);
+				setIsInstalled( true );
 			} )
 			.catch( ( response: { errors: Record< string, string > } ) => {
 				createNoticesFromResponse( response );
 				setInstallingPlugin( null );
 			} );
 	};
 
-	const pluginsList = ( recommendedPlugins || [] ).map(
-		( plugin: Plugin ) => {
+	const pluginsList = ( paymentGatewaySuggestions || [] )
+		.filter( ( plugin: Plugin ) => {
+			return (
+				! installedPaymentGateways[ plugin.id ] &&
+				plugin.plugins?.length &&
+				! plugin.id.startsWith( 'woocommerce_payments' )
```

Comment:
Given the `woocommerce_payments` promotion is still listed as a feature, could we add this condition `window.wcAdminFeatures['wc-pay-promotion']`. Just incase we decide to disable it.

Something is to be said for when we move things out of the feature folder, but that is another discussion to be had :)

Reply:
Nice!  Great idea.  Added.